### ISSUE TITLE
Configs about training mllm-sae in Qwen-2.5-vl-7b

### ISSUE THREAD

**Issue by feimo-49** [EvolvingLMMs-Lab/sae#3]
Thanks for your excellent and insightful work on Multimodal Sparse Autoencoders!

I am currently working on training my own SAE on the Qwen-2.5-VL-7B model, following your methodology in 《Large Multi-modal Models Can Interpret Features in Large Multi-modal Models》. My training runs are taking significantly longer than I anticipated (over 300 hours so far), and I'm trying to ensure my configs are in the right ballpark.

I would be extremely grateful if you could share a few details about your training process, and any information you could provide would be immensely helpful. I'm particularly interested in:

- Training dataset: Is the dataset the whole LLAVA-NEXT-DATA? And does the number of training data need to be over 1M?
- Training epochs and learning rate,
- Number of GPUs and approximate total training time you observed on your hardware.

Thank you again for your groundbreaking research.
